package.json

```diff
@@ -55,7 +55,6 @@
     "gulp": "^4.0.2",
     "gulp-cli": "^2.3.0",
     "gulp-rename": "^2.0.0",
-    "gulp-task-listing": "^1.1.1",
```

Comment:
🔥

Reply:
Moved to PR:

* https://github.com/alphagov/govuk-frontend/pull/3402